charts/kubescape-operator/values.yaml

```diff
@@ -96,6 +96,12 @@ configurations:
   # Enable Prometheus annotations, to allow open-source Prometheus (not operator) to scrape metrics
   prometheusAnnotations: disable
 
+# installation of the alertCRD chart
+alertCRD:
+  installDefault: false # install the default CRD 
+  scopeClustered: false # it is better to have the CRDs in the cluster scope
```

Comment:
We must ensure:
a. they are mutual exclusive
b. no 2 versions of the same CRD at a time